npu.user.js

```diff
@@ -370,6 +370,7 @@ var npu = {
 			$("#panHeader, #panCloseHeader").hide();
 			$("table.top_menu_wrapper").css("margin-top", "5px").css("margin-bottom", "8px");
 			$("#form1 > fieldset").css("border", "0 none");
+			$("#span_changeproject").parent().hide();

```

Comment:
Okay, I did not wish to make a separate commit for this. Long ago a new button (to the menu bar) was added for *Meet Street*, but by removing the top header bar, a "Switch to Meet Street" hyperlink is presented in the top left corner. This removes it.